js_modules/dagit/packages/core/src/instance/BackfillRow.tsx

```diff
@@ -124,18 +134,18 @@ export const BackfillRow = ({
         />
       </td>
       <td style={{width: 140}}>
-        {counts ? (
+        {counts || statusUnsupported ? (
           <BackfillStatusTag backfill={backfill} counts={counts} />
         ) : (
-          <LoadingOrNone queryResult={queryResult} />
+          <LoadingOrNone queryResult={queryResult} noneString="" />
```

Comment:
Could there be some text instead of an empty string? Maybe a `–`, or something that indicates what's missing?

Reply:
Ahh I like that - it looks like we use {'\u2013'} elsewhere for a dash, will use that here!